# Generic message to send code. Used in /submit_code and /submit_guess

string player_name  # name of the player submitting the guess
uint8[4] code       # array of numerical value of colors (e.g., [1, 2, 1, 4])